### ISSUE TITLE
Tracking Central UI 

### ISSUE THREAD

**Issue by swiftdiaries** [kubeflow/kubeflow#199]
Follow up to #146 and addresses #141. 

To do:
- [x] Added as a separate ksonnet component
- [ ] Documentation, how to use
- [ ] E2E tests
- [ ] Release process #527
- [ ] Provide alternative for googleapis.com for MDL.

**Comment by jlewi** [comment]
I suggest filling separate issues for each of these and labeling them area/ui.

We should try to split it up into work items that should take no longer than 2 weeks and ideally can be worked on in parallel.

**Comment by jlewi** [comment]
Closing this in favor of more specific issues with label area/front-end
/close
